app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,8 @@
 class PagesController < ApplicationController
   def home
+  	if current_user && current_user.user_role == User::ROLES[:admin] 
+  		redirect_to admin_users_path
```

Comment:
indentation issue

Reply:
The indentation was fine in my text editor.